Update index.md (#261)
rsksmart/devportal: tutorials/frontend/first-frontend-web3-injected.md

Go to [http://remix.ethereum.org/](http://remix.ethereum.org/)

![Remix Homepage](/assets/img/tutorials/first-frontend-web3-injected/image-15.png)

> NB: Ensure to click the **Solidity** button under **Environment** in the image above, since our code is written in the solidity language, we'll use the Solidity Environment to compile our smart contract.

Reviewer: ```suggestion
Be sure to click the **Solidity** button under **Environment**, as seen in the image above.
We need this in order to compile our smart contract as our code is written in the Solidity language.
```

### Compile a smart contract 

In the 3rd button at the left side select Solidity compiler.

![Solidity Compiler](/assets/img/tutorials/first-frontend-web3-injected/image-14.png)

Click the button **Compile Register.sol**.

![Compile Register.sol](/assets/img/tutorials/first-frontend-web3-injected/image-16.png)

### Deploy a smart contract 

In the left side panel, go to the button Deploy and run transactions. 

Under `Environment`, make sure you have selected the `Injected Web3` option,

> as this tells Remix to use the Web3 provider injected by a browser plugin such as [MetaMask](https://metamask.io/) or [Nifty](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid?hl=en).

Reviewer: Consider using the Nifty wallet homepage instead of the link directly to it's Chrome plugin page, as that could change.
```suggestion
as this tells Remix to use the Web3 provider injected by a browser plugin such as [MetaMask](https://metamask.io/) or [Nifty](https://www.poa.network/for-users/nifty-wallet).
```

![Injected Web3](/assets/img/tutorials/first-frontend-web3-injected/image-13.png)

Then click the button `Deploy`.

![Deploy and run transactions](/assets/img/tutorials/first-frontend-web3-injected/image-05.png)

> > NB: Ensure you have test ETH in your wallet, this is needed to deploy the smart contracts, to do this please follow the guide: [using Remix and Metamask with RSK testnet](/tutorials/ethereum-devs/remix-and-metamask-with-rsk-testnet/).

Reviewer: ```suggestion
> Ensure that you have tR-BTC in your wallet, as this is needed to pay gas fees when deploying the smart contracts. To do so, please follow this guide: [Using Remix and Metamask with RSK testnet](/tutorials/ethereum-devs/remix-and-metamask-with-rsk-testnet/).
```

Click on the **Get** button.

> The web3 instance will call the `setInfo()` function at the smart contract instance `register`, with the info that you entered into the text field.

Reviewer: ```suggestion
The web3 instance will call the `setInfo()` function on the `register` smart contract instance, with the info that you entered into the text field.
```

Enter any value in the info text field, and click on the Set button.

> The web3 instance would call the `setInfo()` function at the smart contract instance register, with the info that you defined.

Reviewer: ```suggestion
The web3 instance will call the `setInfo()` function at the smart contract instance register, with the info that you entered into the text field.
```
